apps/evm/src/clients/api/mutations/borrow/useBorrow.ts

```diff
@@ -4,24 +4,17 @@ import { type UseSendTransactionOptions, useSendTransaction } from 'hooks/useSen
 import { useAnalytics } from 'libs/analytics';
 import { useGetNativeTokenGatewayContract, useGetVTokenContract } from 'libs/contracts';
 import { useChainId } from 'libs/wallet';
-import type { VToken } from 'types';
+import type { Pool, VToken } from 'types';
 import { callOrThrow, convertMantissaToTokens } from 'utilities';
 
 type TrimmedBorrowInput = Omit<BorrowInput, 'vTokenContract' | 'nativeTokenGatewayContract'>;
 type Options = UseSendTransactionOptions<TrimmedBorrowInput>;
 
-const useBorrow = (
-  {
-    vToken,
-    poolName,
-    poolComptrollerAddress,
```

Comment:
I'd keep these two props rather than requiring a pool to be passed. We only need two properties from the `pool` object, requiring the whole object to be passed makes the hook a bit harder to use imo.

Reply:
I went back for the individual props, but made them mandatory so we avoid silently failing like before